oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,146 @@
+
+#. An existing CC ("sponsor") choses to nominate a community member ("candidate") for a new CC position
+#. The sponsor posts a new public thread on Discourse, in the `Core Contributor
+   discussion space <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_, that
+   introduces the candidate, describes in a few paragraphs why they are suited to
+   join the program, and provides links to examples of previous Open edX work (for example, pull requests).
+   The post must mention the date when the comment period ends, which is one month from the start of the thread.
+#. The sponsor posts a link to the Discourse post in the ``#core-contributors`` Slack room,
+   as well as any other Slack rooms relevant to the CC role being discussed. The message must indicate
+   the dates of the comment period.
+#. During the comment period, other CCs should weigh in on the thread. If a CC has concerns they feel
+   cannot be raised publicly, they should consult with the Program Administrator to determine how to
+   proceed. Remember to leave constructive criticism - if a CC candidate is not ready yet, provide some
+   tips as to how they could make it into the program.
+#. Over the course of the comment period, the sponsor should respond to all comments that come in. It
+   may also be necessary to periodically nudge other CCs to respond to the thread.
+#. At the end of the comment period, if the candidate is approved, the candidate should work with the Program
+   Administrator to get the access they need for their role. If concerns have been brought up, the sponsor
+   should meet with the candidate to discuss what they need to work on to become a CC. We enourage the
+   sponsor to help the candidate develop a plan of action so they can be a successful candidate in the future.
+   
+Where Do I Start?
+^^^^^^^^^^^^^^^^^
+
+New community members interested in the Core Contributor program should `get in touch with us! <https://open.edx.org/community/connect/>`_
+The ``#core-contributors`` room in Slack can help guide people towards people working in their
+area(s) of interest.
+
+Existing community members who have a record of contributing to the Open edX project should feel free
+to reach out to `current CCs <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Current+Open+edX+Core+Contributors>`_
+who have the role they're interested in. Engage in conversation to see what it's
+like to be that type of CC; you might ask them to take a look at your work, to see if they would be
+able to sponsor you for inclusion into the CC program. CCs may be pretty overloaded, so don't take
+offense if the first person you reach out to doesn't have time at that moment to help. And if you don't
+know where to begin, find us in ``#core-contributors`` on Slack or visit us at the next
+`Contributor's Meetup <https://calendar.google.com/calendar/embed?src=c_mvh2oito7s9n89g5nnkk49psks%40group.calendar.google.com&ctz=America%2FNew_York>`_.
```

Comment:
The core contributors sprints could also be a good place to come get a task assigned? Or the individual workgroups meetings.

Reply:
Hmm. I think I want to not enshrine specific meetings in this doc, as OEPs aren't generally updated frequently, and meeting purpose/content (or the meeting itself!) may change over time. I think I'll remove the meetup calendar and point at the CC forum topics.